drivers/bluetooth/hci/h5.c

```diff
@@ -465,6 +466,15 @@ static void bt_uart_isr(const struct device *unused, void *user_data)
 				break;
 			}
 
+			header_checksum = ~(hdr[0] + hdr[1] + hdr[2]);
```

Comment:
Consider moving this calculation into a function since it is used multiple times

Reply:
I don't think so. This is a tiny operation. The function involves parameter copy and stack pressing and out of stack operations. An inline may be added. Compiler optimization will ask us to do this. Maybe defining a macro is more direct.

Reply:
A macro would also be suitable; anything that can ensure that this calculation is done one way, and one way only.